config/routes.rb

```diff
@@ -431,7 +431,7 @@
 
   mount TestUserDashboard::Engine, at: '/test_user_dashboard' if Settings.test_user_dashboard.env == 'staging'
 
-  if Rails.env.development? || Settings.vsp_environment == 'development'
+  if Rails.env.test? || Rails.env.development? || Settings.vsp_environment == 'development'
```

Comment:
Should we add some specs around these conditions? Mainly checking that this wont be mounted `staging`, `sandbox`, and `prod`. Just in case something inadvertently gets changed

Reply:
What I recommended is (if we want to do this) that we make a datadog test that just attempts to hit the route and proves that it can't be hit. Seems to me like an rspec test for this would be kinda contrived, but a 'live' test would truly prove that we couldn't reach the route